Tests/XcodeProjTests/Scheme/XCSchemeTests.swift

```diff
@@ -655,6 +660,7 @@ final class XCSchemeIntegrationTests: XCTestCase {
         XCTAssertEqual(scheme.profileAction?.buildConfiguration, "Release")
         XCTAssertTrue(scheme.profileAction?.shouldUseLaunchSchemeArgsEnv == true)
         XCTAssertEqual(scheme.profileAction?.savedToolIdentifier, "")
+        XCTAssertNil(scheme.profileAction?.customWorkingDirectory)
```

Comment:
It would be great to update one of the fixtures to include a `customWorkingDirectory` such that one of the tests verifies serialising / deserialising it